src/commands/misc/help.js

```diff
@@ -0,0 +1,33 @@
+const { EmbedBuilder } = require('@discordjs/builders')
+
+module.exports = {
+    name: 'help',
+    description: 'List of commands',
+
+    callback: (client, interation) => {
+        
+        const embed = new EmbedBuilder()
+        .setTitle('Server Commands')
+        /*.setDescription('List of server commands')*/
+        .addFields(
+            {
+                name: '/help-remember',
```

Comment:
I'm usnure if any of the fields here are neccessary besides '/help-remember' as the rest are debug commands. You can add another field for reminder features as those aren't stated anywhere